### ISSUE TITLE
TypeError: import_NameManager.default is not a constructor

### ISSUE THREAD

**Issue by hseoy** [nihgwu/react-runner#108]
I think this is wonderful. However, due to the error below, it cannot be applied to my website.

```
TypeError: import_NameManager.default is not a constructor
    at getSucraseContext (index.mjs:93:23)
    at transform (index.mjs:40:28)
    at p (transform.ts:4:10)
    at f (utils.ts:28:12)
    at getDerivedStateFromProps (Runner.ts:37:18)
    at applyDerivedStateFromProps (react-dom.development.js:12432:22)
    at mountClassInstance (react-dom.development.js:12871:5)
    at updateClassComponent (react-dom.development.js:17426:5)
    at beginWork (react-dom.development.js:19073:16)
    at beginWork$1 (react-dom.development.js:23940:14)
```

**Comment by nihgwu** [comment]
Which build tool are you using? But I guess it's Vite/esbuild, checkout [this](https://github.com/nihgwu/react-runner/blob/master/examples/with-vite/vite.config.ts#L10), and the [reason](https://github.com/vitejs/vite/issues/7112#issuecomment-1065130809)
I think I should add a Troubleshooting section to readme

**Comment by hseoy** [comment]
I'm using a [remix](https://remix.run/). Is there a problem if it's not a Vite?

**Comment by nihgwu** [comment]
@hseoy Should be the same issue https://github.com/FormidableLabs/react-live/issues/306#issuecomment-1075173629

**Comment by nihgwu** [comment]
@hseoy I just released 1.0.1, with that version, it should just work for Remix
